api-reference/10 UI Components/DataExpressionMixin/1 Configuration/valueExpr.md

```diff
@@ -14,7 +14,9 @@ The current item's data object.
 A unique item identifier.
 
 ---
-Always set this property if the [dataSource](/api-reference/10%20UI%20Components/DataExpressionMixin/1%20Configuration/dataSource.md '{basewidgetpath}/Configuration/#dataSource') contains objects. Alternatively, you can specify the [key](/api-reference/30%20Data%20Layer/Store/1%20Configuration/key.md '/Documentation/ApiReference/Data_Layer/CustomStore/Configuration/#key') in the store. 
+When you do not set this property, the **value** contains a data item from [dataSource]({basewidgetpath}/Configuration/#dataSource) as it is. 
```

Comment:
Why do we decide to remove the recommendation to use this option for item-objects? Given past support traffic - it's not obvious to everyone that references to objects (that are equal by fields value) can be different.